apps/accounts/views/upstream_login.py

```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render
+from django.conf import settings
+import urllib.request as req
```

Comment:
try `from urllib.request import pathname2url` for clarity?

Reply:
@samgensburg-gov We aren't using the upstream_login view, but I left it because its the basis for supporting an upstream OAuth2 provider.  I can remove...let me know.

Reply:
If we're not using it, let's remove it.